### ISSUE TITLE
Edge post filter issue

### ISSUE THREAD

**Issue by medion6** [SuiMachine/Deadly-Premonition-2---Sui-s-hack#11]
The edge detection post filter creates lines on characters' faces in close-ups or when you zoom in the camera too close.

**Comment by SuiMachine** [comment]
Edge detection filter is a game feature that works by default, this hack only allows to disable it.
I think you are talking about HBAO instead, which on higher profiles will make edges more visible (since it seems to consider surface normals and now just depth). There is not much that can be done about it, due to how low-poly these character models are (and badly rigged as well).

**Comment by medion6** [comment]
The lines only appear for me with the edge detection filter on. HBAO work just fine.

**Comment by SuiMachine** [comment]
Ok, this is certainly Edge detection filter. But I don't exactly see what could be causing it. I have RTX 3070 (driver 516.59, but without Nvidia's GeForce Experience) and 2K screen - no issues like this. I even played around with some Nvidia settings seeing if I could break it. 

I have seen things like that happening in the game I work on, but that was due to some buffers that were used in post-process being scaled incorrectly and that's not exactly something that anyone expect game developers should have access to.

**Comment by medion6** [comment]
Hi Sui suit just solved the issue magically with a restart and another complete clean install of the mod and game (?¿). 
Which is even weirder because it's not the first reinstall I've done to see if it fixes it...
The only thing I can think of is that I previously had a 4k DSR option checked in the Nvidia panel but I wasn't using it, my desktop was at native resolution and the game too in theory.

**Comment by medion6** [comment]
I already know where the problem is. Touching the far clip option up causes the issue.

**Comment by SuiMachine** [comment]
Thanks for letting me know. To an extend it does make sense - I will see if something can be done with it (although probably it will have to be next month).

**Comment by SuiMachine** [comment]
Right. I played around a bit with it and there is Edge Detection Depth setting in the filter, which I exposed. If you extend camera's far clip distance above 800 units, an slider will be displayed under edge filter now telling you why it's there.
